tools/pipelines/test-real-service.yml

```diff
@@ -78,11 +78,12 @@ jobs:
       inputs:
         command: 'custom'
         workingDir: packages/test/end-to-end-tests
-        customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
+        customCommand: 'run ci:test:realsvc'
```

Comment:
Is this script updated in another PR?

Reply:
i removed exit 0 already, build is broken